src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -1158,34 +1171,42 @@ public async Task GetRequestStreamAsync_SetPOSTThenGet_ExpectNotNull()
             });
         }
 
+        [Fact]
         public async Task GetResponseAsync_GetResponseStream_ExpectNotNull()
         {
             await LoopbackServer.CreateServerAsync(async (server, url) =>
             {
                 HttpWebRequest request = WebRequest.CreateHttp(url);
-                using (WebResponse response = await request.GetResponseAsync())
+                Task<WebResponse> getResponse = request.GetResponseAsync();
+                await server.HandleRequestAsync();
+                using (WebResponse response = await getResponse)
+                using (Stream myStream = response.GetResponseStream())
                 {
-                    Assert.NotNull(response.GetResponseStream());
+                    Assert.NotNull(myStream);
                 }
             });
         }
 
-        [Theory, MemberData(nameof(EchoServers))]
-        public async Task GetResponseAsync_GetResponseStream_ContainsHost(Uri remoteServer)
+        [Fact]
```

Comment:
Same http/https comment.

I'll stop commenting on these.  They apply to most of these tests.